lib/include/vehicle.h

```diff
@@ -1,14 +1,14 @@
 #pragma once
 
 #include <cstdint>
+#include "transportation_quantity.h"
 
 namespace vrp {
 /// Vehicle representation
 class Vehicle {
 public:
     int id = 0;  ///< vehicle id
-    int capacity = 0;  ///< vehicle capacity
-    int max_weight = 0;  ///< max vehicle weight
+    TrQuant trquant_v; ///vehicle quantity
```

Comment:
I'd simply name `quantity`

Reply:
It should be: TransportationQuantity capacity
And fields capacity and max_weight should be removed

Reply:
agree with Mike